### PR TITLE
Moving code to Cypress 10

### PR DESCRIPTION
**Pull request by nadvolod**
I moved all of our code to Cypress 10. It was very easy if I didn't have to update all of our tests and simply could use what Cypress maintains.

My question is do we even need this any more? The dev team is doing a wonderful job of maintaining https://github.com/saucelabs/saucectl-cypress-example . Maybe we can just add a few features there as we need.

### CODE REVIEW

File: testrunner-toolkit/cypress/cypress.config.js
Comment (wswebcreation) [inline testrunner-toolkit/cypress/cypress.config.js:6]: Can you provide an example? If not, then please add a link that will go to the Cypress docs on how to use this

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Thanks for the PR. I've added a few inline comments and I also have a few other comments:

- It now looks like we dropped support for 9 and lower. Can you add two examples @nadvolod  where we support before and after Cypress 10?
- I assume you copied the examples from the Cypress repo, then please also give some credits to it somewhere in the files, with a reference, and in the readme
- I'm not a big fan of using all those examples, this means double code (maybe still a simple copy action from the Cypress repo, but still.)  For me this doesn't add any value in comparison to the Cypress samples, it also doesn't really focus on showing how to use Cypress with Sauce Labs and that's why I think we should prevent this. The samples that we have in other repos are more examples on:
  - how to run specific actions on Sauce
  - how to automate certain things
- why do we have a screenshot in this repo
- I'm missing updates to both README.MD files, can you also make sure they are updated with the correct info?

Thanks!

**Comment by eyaly** [comment]
We can NOT merge our code the Cypress 10. 
We can create a separate folder for cypress 10 as in the product repo.
SAP teams (and I assume many more customers) are still using Cypress < 10 and we need this repo to demo them on Cypress 9.

Also @nadvolod - Please wait for the changes after we merge PR 222 But, again, the need here is to create a separate folder or project and leave Cypress 9 examples with my changes :-)

**Comment by nadvolod** [comment]
@eyaly @wswebcreation seems like you all answered my question with so many quests. How about I just create a cy-10 folder with a link to https://github.com/saucelabs/saucectl-cypress-example where Cypress 10 examples are maintained by the product team? And they do a very good job.

**Comment by wswebcreation** [comment]
lol, yes, you can do that, then please add a note to the readme that the js-config file is now the way to go instead of the json config gile.

**Comment by eyaly** [comment]
@nadvolod 
How about I just create a cy-10 folder with a link to https://github.com/saucelabs/saucectl-cypress-example where Cypress 10 examples are maintained by the product team? 

For me, it is a workaround until we will have time to add our examples. 
The product team repo has one example that covers running on Sauce, running on Docker on 2 browsers.
This example is great for our documentation and sending to customers but not for demos.
For me, to demo how to run Cypres on Sauce, I need more examples.

**Comment by wswebcreation** [comment]
Hi @nikolay-advolodkin 

What's the status of this PR, can we close this one and wait for a new one?

**Comment by wswebcreation** [comment]
Closing this due to inactivity

**Pull request closed without merging**
